client/blocks/taxonomy-manager/list.jsx

```diff
@@ -68,15 +57,12 @@ export class TaxonomyManagerList extends Component {
 		}
 
 		// if item has a parent, and parent is in payload, height is already part of parent
-		if ( item.parent && ! _recurse && includes( this.termIds, item.parent ) ) {
+		if ( item.parent && ! _recurse && this.props.termIds.includes( item.parent ) ) {
```

Comment:
You can remove `termIds` completely and write:
```js
this.props.terms.some( ( term ) => term.ID === item.parent )
```

Reply:
Much better 👍🏻 changed in https://github.com/Automattic/wp-calypso/pull/57819/commits/4f7fc90944c6f0ea18caf78534a5ddc000169278